user/urls.py

```diff
@@ -1,8 +1,10 @@
-from django.urls import path
+from django.urls import path, include
 
 from . import views
 
 urlpatterns = [
     path('', views.TopView.as_view(), name='top'),
     path('signup', views.SignupView.as_view(), name='signup'),
+    path('', include('django.contrib.auth.urls')),
```

Comment:
下記が望ましいです。[参照](https://docs.djangoproject.com/en/4.0/topics/auth/default/#module-django.contrib.auth.views)
```suggestion
    path('accounts', include('django.contrib.auth.urls')),
```